package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ansible-language-server",
+  "name": "@ansible/ansible-language-server",
```

Comment:
@relrod and @cidrblock — this is another option:
```suggestion
  "name": "@ansible/language-server",
```

Basically, this affects:
1) The URL under which it's listed on npmjs.org
2) How it's installed (what people type — `npm i @ansible/ansible-language-server` vs `npm i @ansible/language-server`
3) What people add to the dependencies in their `package.json`
4) How people import it for use in their projects

Once you're available, please either comment with your concerns or approve this PR.

Reply:
+1 for @ansible/ansible-language-server

Reply:
+1 for @ansible/ansible-language-server